File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 6/SOAL 2/PRAK602-2410817210002-Sebastian_Abe_Santoso.py
def zetsu(b, n):
    for i in range(n):
        b[i] = b[i] * (i + 1)
ruang = int(input(""))
b = list(map(int, input().split()))
if len(b) != ruang:
    print("Error: Nomor diatas batas")
else:
    zetsu(b, ruang)
    for i in range(ruang):
        print(b[i], end=' ')

File: MODUL 6/SOAL 2/PRAK602-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
void zetsu(int b[], int n) {
    for (int i = 0; i < n; i++) {
        b[i] = b[i] * (i + 1); // 
    }
}
int main() {
    int ruang;
    scanf("%d", &ruang);
    int b[ruang];
    for (int i = 0; i < ruang; i++) {
        scanf("%d", &b[i]);
    }
    zetsu(b, ruang);
    for (int i = 0; i < ruang; i++) {
        printf("%d ", b[i]);
    } printf("\n");
    return 0;
}